packages/eslint-config-godaddy-minimal/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  extends: [
+    require('./extends')('eslint-config-godaddy-react-flow')
+  ],
+  rules: {
+    'complexity': 0,
+    'consistent-return': 0,
+    'id-length': 0,
+    'indent': [2, 2, {
+      VariableDeclarator: { var: 2, let: 2, const: 3 },
+      flatTernaryExpressions: true,
+      MemberExpression: 'off'
+    }],
+    'max-params': 0,
+    'max-nested-callbacks': 0,
+    'max-statements': 0,
+    'no-else-return': 0,
+    'no-nested-ternary': 0,
+    'no-shadow': 0,
+    'no-undefined': 0,
+    'no-unused-vars': ['error', { args: 'none', ignoreRestSiblings: true }],
+    'spaced-comment': 0
```

Comment:
If we don't remove this rule, can it just be one that gets automatically fixed like some other formatting is, or a warning? I have never understood why such a trivial thing is a linting `error`.  And it is so annoying when you just want to comment out several lines while actively testing code to have the linter error because of a space.